caffe2/quantization/server/fully_connected_dnnlowp_op.h

```diff
@@ -62,4 +68,884 @@ class FullyConnectedDNNLowPOp
   std::int32_t in_qparams0_zero_point_old_ = 0;
 }; // class FullyConnectedDNNLowPOp
 
+using namespace std;
```

Comment:
At minimum, we shouldn't put this in a header file.

Reply:
Right, I'll remove it.